internals/utils/src/race-async-generators.ts

```diff
@@ -1,3 +1,4 @@
+// TODO (Ethan-Arrowood) - Potential dead code
```

Comment:
no rush to delete this code since it isn't causing problems at this time. I'll circle back to these todo's at a later point as i imagine theres more dead code throughout the code base i can eliminate